### PR TITLE
Knadeem added cities

### PR DESCRIPTION
**Pull request by knadeem11**
added centreville and atlanta cities

### CODE REVIEW

File: Cities.txt [hunk 1]
```diff
@@ -3,3 +3,5 @@ New Jersey
 Washington DC
 Miami
 San Francisco
+Atlanta
+Centreville
```
Comment (qnadeem1) [inline Cities.txt]: Is Centreville a city?

File: Cities.txt [hunk 2]
```diff
@@ -3,3 +3,5 @@ New Jersey
 Washington DC
 Miami
```
Comment (qnadeem1) [inline Cities.txt:4]: What about adding LA instead of Centreville?

**Review by qnadeem1 — COMMENTED** [review]
please review my comments

File: Cities.txt (hunk 2 above)
Comment (knadeem11) [reply Cities.txt:4]: sounds good

**Pull request merged**